include/cglm/vec3.h

```diff
@@ -694,9 +694,9 @@ glm_vec3_center(vec3 a, vec3 b, vec3 dest) {
 CGLM_INLINE
 float
 glm_vec3_distance2(vec3 a, vec3 b) {
-  return glm_pow2(b[0] - a[0])
-       + glm_pow2(b[1] - a[1])
-       + glm_pow2(b[2] - a[2]);
+  return glm_pow2(a[0] - b[0])
+       + glm_pow2(a[1] - b[1])
+       + glm_pow2(a[2] - b[2]);
```

Comment:
May I ask what is the benefit[s] of using `a` first instead of b?

Reply:
I used `b - a` here because the distance formula is like `sqtr(pow(x2 - x1, 2) + pow(y2 - y1, 2) + ...)`